### PR TITLE
CAT-FIX crash report dialog

### PR DESCRIPTION
**Pull request by jonny911**

### CODE REVIEW

File: catroid/build.gradle [hunk 1]
```diff
@@ -59,6 +60,7 @@ android {
             buildConfigField "boolean", "FEATURE_ARDUINO_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_RASPI_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_POCKETMUSIC_ENABLED", "true"
+            buildConfigField "boolean", "FIREBASE_CRASH_REPORT", "false"
```
Comment (fwinkelbauer) [inline catroid/build.gradle]: Is there a good reason that this flag should be named different than the others? If not, please change it to something like `FEATURE_FIREBASE_CRASH_REPORT`

Comment (jonny911) [inline catroid/build.gradle]: It's not considered to be a feature...

Comment (thmq) [inline catroid/build.gradle]: In the spirit of consistency you should consider naming it `FIREBASE_CRASH_REPORT_ENABLED`. 😼

Comment (fwinkelbauer) [inline catroid/build.gradle]: :+1:

**Pull request merged**